FreeAPS/Sources/APS/OpenAPS/OpenAPS.swift

```diff
@@ -635,7 +635,7 @@ final class OpenAPS {
 
     private func middlewareScript(name: String) -> Script? {
         if let body = storage.retrieveRaw(name) {
-            return Script(name: "Middleware", body: body)
```

Comment:
Traced it down to here.

Once changed this is the output:
```
OpenAPS.swift - determineBasal(currentTemp:clock:) - 21 DEV: Start determineBasal
JavaScriptWorker.swift - outputLogs() - 78 DEV: middleware/determine_basal.js: Middleware reason: Nothing changed
JavaScriptWorker.swift - outputLogs() - 78 DEV: middleware/determine_basal.js: Pumphistory is empty!
JavaScriptWorker.swift - outputLogs() - 78 DEV: middleware/determine_basal.js: insulinFactor set to : 55
JavaScriptWorker.swift - outputLogs() - 78 DEV: middleware/determine_basal.js: Using weighted TDD average: 1 U, instead of past 24 h (0 U), weight: 0.65
JavaScriptWorker.swift - outputLogs() - 78 DEV: middleware/determine_basal.js: , Dynamic ratios log: Dynamic Settings disabled
```

Reply:
@marionbarker, @dnzxy & @mountrcg : let me know what you think of this.

Reply:
I think this is good. ✔️